aptos-move/aptos-e2e-comparison-testing/src/online_execution.rs

```diff
@@ -0,0 +1,240 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    compile_aptos_packages, dump_and_compile_from_package_metadata, is_aptos_package,
+    CompilationCache, ExecutionMode, IndexWriter, PackageInfo, TxnIndex, APTOS_COMMONS,
+};
+use anyhow::Result;
+use aptos_framework::natives::code::PackageMetadata;
+use aptos_language_e2e_tests::data_store::FakeDataStore;
+use aptos_rest_client::Client;
+use aptos_types::transaction::Version;
+use aptos_validator_interface::{AptosValidatorInterface, FilterCondition, RestDebuggerInterface};
+use move_core_types::account_address::AccountAddress;
+use std::{
+    collections::HashMap,
+    path::PathBuf,
+    sync::{Arc, Mutex},
+};
+use url::Url;
+
+pub struct OnlineExecutor {
```

Comment:
Are we going to use online executor for CI testing? Will that be too slow?

Reply:
It is for debugging and will not be used on CI